setup.py

```diff
@@ -5,67 +5,96 @@
 # All rights reserved.
 #
 
-import os,sys
+import os
+import sys
 from os.path import expanduser
 from setuptools import setup, find_packages
 from setuptools.command.install import install
 import ConfigParser
 
+
 class OverrideInstall(install):
-   
     def run(self):
-        
+
         for arg in sys.argv:
             if '--install-data' in arg:
                 break
         else:
-            self.install_data = '/etc/jsnapy'
-            
+            if hasattr(sys, 'real_prefix'):
+                self.install_data = os.path.join(os.environ.get('VIRTUAL_ENV'),
```

Comment:
I think it would be good to have the location of test and config yamls to be static. It shouldn't happen that a user is forced to copy his files when he changes his virtualenv. Changing a virtaulenv should be treated as upgrade/downgrade on the fly. Normally when we upgrade/downgrade we don't need to restore our test/config files.

os.environ.get('VIRTUAL_ENV') won't work when virtualenv is not activated and user runs with virtualenv python binary directly.  So use sys.prefix if you have to.